economic_analysis/Facades Economic Analysis.py

```diff
@@ -0,0 +1,155 @@
+#import packages
+import pandas as pd
+import numpy as np
+import os
+
```

Comment:
@ekpresent See comments below, there is redundant code that can be condensed, and potentially areas to improve speed if that is a concern, but overall I think the approach works.